m4_sessions/1811_inning/assertiveness.md

```diff
@@ -1,30 +1,36 @@
 ## Group Members:
+* Kevin Simpson
+* Daniel Mulitauopele
+* Alex Ela
 
-
-
-## Name of Session: 
+## Name of Session: Developing Healthy Assertiveness in the Workplace
 
 ## Session Description 
 
-(Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
+Do you have trouble voicing your opinion at crucial moments? Ever wanted a raise, but didn't know how to ask? Don't know how to ask your co-worker to turn the music down? Come to the Healthy Assertiveness Gear-Up.
 
 ## Session Outcomes 
 
-(What will students learn/leave with?): 
+* For students to feel more comfortable/confidence asserting themselves than they were when they walked in
+* For students who don't have a problem with being assertive finding ways to help others find their voices
 
 ### Recommended Trajectory 
 
-     Best practices:  No section should be longer than 20 minutes
-     Create a session that YOU would want to attend
-     Share resources in your slack channel
-     All sessions NEED to have a closing activity, reflection, discussion
-     # Introductons 
-     # Intro to new material
-     # Small Group Activity
-     # Closing Discussion
+* What does "assertive" mean to you?
+* What are good practices for being assertive?
+* What are bad practices for being assertive?
 
 ## Session Outline (Agenda for session, include timing for each session and who is facilitating)
 
+* Intro - why did you select this session, on a scale of 1-10, how assertive do you think you are?
+* Journal - when did your assertiveness (or lack of) play a role in an outcome?
+* Share journals by volunteer
+* Warm-up - physical/vocal
+* Leaders role-play a salary discussion with feedback
+* Small group-based role-play (low-stakes)
+* Volunteers for demonstrate advanced assertiveness
+* Reflection journaling
```

Comment:
What is the final reflection?  How can someone continue to improve their assertiveness?